portfolio/src/main/java/com/google/sps/classes/Comment.java

```diff
@@ -12,15 +12,16 @@
   private String comment;
   private String date;
 
-  public Comment(String c, String n){
-    name = n;
-    comment = c;
+
+  public Comment(String commentIn, String nameIn){
+    name = nameIn;
```

Comment:
This can simply be

this.name = name;

It doesn't necessarily have to be nameIn.